### PR TITLE
[11.0][MIG] procurement_auto_create_group

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration to v11. Adapted to work without procurement orders.

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM :+1:

File: procurement_auto_create_group/README.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+#. Go to *Inventory / Configuration / Settings* and check the option 'Advanced
```
Comment (JordiBForgeFlow) [inline procurement_auto_create_group/README.rst]: #. Go to *Inventory / Configuration / Settings* and check the option 'Multi-Step Routes' and press the 'Save' button.

File: procurement_auto_create_group/README.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   routing of products using rules' and press the 'Apply' button.
+#. Activate the developer mode.
+#. Go to *Inventory / Configuration / Routes / Routes* and check the option
```
Comment (JordiBForgeFlow) [inline procurement_auto_create_group/README.rst]: Replace this with:
#. Go to *Inventory / Configuration / Warehouse Management / Routes* and select the route you want to change. Select the pull rule you wish to change and Select 'Propagation of Procurement Group': 'Propagage'. The checkbox 'Auto-create Procurement Group' will then appear and you can set it if you want to procurement group to be automatically created.

File: procurement_auto_create_group/README.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+   'Auto-create Procurement Group' to the pull rules where you want the
+   procurement groups to be automatically proposed.
+
+Usage
+=====
+
+#. Create a new procurement and make sure that it determines a pull rule
+   with the option 'Auto-create Procurement Group' set.
+#. When you save the procurement order, a procurement group with format
```
Comment (JordiBForgeFlow) [inline procurement_auto_create_group/README.rst]: Replace with:
#. When the procurement rule is executed, a procurement group with format 'PG/000001' will be created.

(as there are no more procurement orders)

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Tested. Works as expected. Minor comments.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@lreficent Can you attend comments ?

File: procurement_auto_create_group/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply procurement_auto_create_group/README.rst]: Thanks for fixing :smiley:

**Comment by LoisRForgeFlow** [comment]
@rousseldenis I think @jbeficent already did it! Feel free to click the green button :wink:

**Comment by rousseldenis** [comment]
@lreficent Done :smile:

**Pull request merged**